api.js

```diff
@@ -165,9 +155,25 @@ Api.prototype._prefixTitle = function (file) {
 Api.prototype.run = function (files) {
 	var self = this;
 
+	if (!files || files.length === 0) {
+		this.files = [
+			'test.js',
+			'test-*.js',
+			'test'
+		];
+	} else {
+		this.files = files;
+	}
+
+	this.excludePatterns = [
+		'!**/node_modules/**',
+		'!**/fixtures/**',
+		'!**/helpers/**'
+	];
+
 	this._reset();
-	this.explicitTitles = Boolean(files);
-	return handlePaths(files || this.files, this.excludePatterns)
+	this.options.explicitTitles = Boolean(files);
+	return handlePaths(this.files, this.excludePatterns)
```

Comment:
Since `files` is provided to the watcher you no longer need to assign it to `this.files`. Just use the variable.